src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -872,7 +878,52 @@ def forward(
         output_hidden_states=None,
         return_dict=None,
     ):
+        r"""
+        labels (:obj:`torch.LongTensor` of shape :obj:`(batch_size, sequence_length)`, `optional`):
+            Labels for computing the masked language modeling loss. Indices should be in ``[-100, 0, ...,
+            config.vocab_size]`` (see ``input_ids`` docstring) Tokens with indices set to ``-100`` are ignored
+            (masked), the loss is only computed for the tokens with labels in ``[0, ..., config.vocab_size]``
+
+        Returns:
+
+        Examples::
 
+            >>> from transformers import LayoutLMTokenizer, LayoutLMForMaskedLM
+            >>> import torch
+
+            >>> tokenizer = LayoutLMTokenizer.from_pretrained('microsoft/layoutlm-base-uncased')
+            >>> model = LayoutLMForMaskedLM.from_pretrained('microsoft/layoutlm-base-uncased')
+
+            >>> words = ["Hello", "[MASK]"]
+            >>> normalized_word_boxes = [637, 773, 693, 782], [698, 773, 733, 782]
+            >>> inputs = tokenizer(words, return_tensors="pt")
+            >>> input_ids = inputs['input_ids']
+            >>> attention_mask = inputs['attention_mask']
+            >>> token_type_ids = inputs['token_type_ids']
+
+            >>> tokens = []
+            >>> token_boxes = []
+            >>> for word, box in zip(words, normalized_word_boxes):
+            ...    word_tokens = tokenizer.tokenize(word)
```

Comment:
can't we do directly `word_ids = tokenizer(word)` here? Don't really want to incentivize the user too much to use `tokenizer` and `convert_tokens_to_ids`

Reply:
Updated this :)